### DISCUSSION TITLE
recent compilation exception

### DISCUSSION THREAD

**Discussion by PaulNext** [vercel/community#2202] [Help] [answered]
### Summary

The same code was successfully deployed before 19h, but the last few commits did not change, but some files were missing, Failed to compile.
./src/lib/hooks/index.ts
Module not found: Default condition should be last one
But there is this file in the actual warehouse, and the local can also be compiled and built successfully. I have worked hard to check the compilation of this piece, thank you!

### Additional information

The local can be successfully compiled, but the error "Module not found: Default condition should be last one" has been reported on the vercel remote machine. This problem has only occurred in the past 24 hours. There have been no major changes recently, even the same commit the day before yesterday (deployment success) and now re-deployment also fails

_No response_

**Comment by whalefrat** [comment, marked as answer]
Solved, here is how i found out what was going on. I searched a lot about "Module not found: Default condition should be last one" error, and when i saw something about ["firebase" issue](https://stackoverflow.com/questions/75337593/react-module-not-found-error-default-condition-should-be-last-one) , I suddenly thought that although my project does not contain firebase, there still may be a dependency package version conflict. Then I checked the version of each dependent package in the previously successful deployment in vercel, and compared it with the version of those in failed deployment. It was "alchemy-sdk@2.8.1" for my deployment failure, "alchemy-sdk@2.8.0" would work.
